snoop/data/models.py

```diff
@@ -66,6 +66,13 @@ class Blob(models.Model):
     def __str__(self):
         return self.pk
 
+    @property
+    def content_type(self):
+        if self.mime_type.startswith('text/'):
+            return f"{self.mime_type}; charset={self.mime_encoding}"
+
+        return self.mime_type
```

Comment:
why not always include the `charset`? or include it when it isn't `binary`?

Reply:
I think it makes no sense unless the content is text. Do you have an example of a non-text mime type that has a charset? Or some documentation somewhere?